javascript/javascript2/week1/shortestWord.js

```diff
@@ -0,0 +1,12 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium", "å"];
+
+function shortestWord(wordList) {
+  let shortWord = wordList[0];
+  for (let i = 1; i < wordList.length; i++) {
```

Comment:
Does array count start with 1 or 0? Can you verify that?  you can console.log " i " and check.

Reply:
it starts with 1; because i considered first word of array as the shortest , compared and swapped with it if shortest word found in the array.

Reply:
Yes, that makes sense. Good job